integrations/tensorflow/e2e/keras/applications/applications_benchmark.py

```diff
@@ -0,0 +1,120 @@
+# Lint as: python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate artifacts for benchmarking models in tf.keras.applications."""
+
+import os
+
+from absl import app
+from absl import flags
+from iree.tf.support import tf_benchmark_utils
+import tensorflow.compat.v2 as tf
+
+FLAGS = flags.FLAGS
+
+# Testing all applications models automatically can take time
+# so we test it one by one, with argument --model=MobileNet
```

Comment:
We always test MobileNet? I'm confused

Reply:
Copied from `_test.py` but forgot to update here.